createBoard.js

```diff
@@ -1,20 +1,30 @@
- 
-function createBoard(x,y,gridType) {
+function createBoard(modelGrid,gridType) {
     var table = '';
-    var row = x;
-    var col = y;
+    var row = modelGrid.length;
```

Comment:
You should make this a single variable now and refactor the row/col out of the loops.